nemo_skills/pipeline/utils.py

```diff
@@ -468,6 +468,8 @@ def get_env_variables(cluster_config):
     - `required_env_vars` - list of required environment variables
     - `env_vars` - list of optional environment variables
 
+    NVIDIA_API_KEY and OPENAI_API_KEY are always added if they exist.
```

Comment:
Can you please also add HF_TOKEN here and remove explicit setting of those vars from https://github.com/Kipok/NeMo-Skills/blob/main/nemo_skills/pipeline/utils.py#L98 and https://github.com/Kipok/NeMo-Skills/blob/main/nemo_skills/pipeline/utils.py#L158

Reply:
Updated in 20ebda1331ddfbf06e3675c19fe252c71c4b2ff9